packages/react-compass/src/accordion/accordion-title.tsx

```diff
@@ -47,20 +50,14 @@ const AccordionTitle = React.forwardRef<HTMLButtonElement, AccordionTitleProps>(
 
     const handleKeyDown = (e?: unknown) => {
       const event = e as React.KeyboardEvent<HTMLElement>
-      const {key} = event
-      switch (key) {
-        case 'Spacebar':
+      const {code} = event
+      switch (code) {
+        case 'Space':
```

Comment:
Move event key out to constants.ts, KeyBoard. vairable name should not be code, it should be key